src/styles/index.scss

```diff
@@ -1,3 +1,107 @@
-body {
+$days: 31;
+$month-lengths: (28, 29, 30, 31);
+$days-month: (
+  mon: 'Monday',
+  tue: 'Tuesday',
+  wed: 'Wednesday',
+  thu: 'Thursday',
+  fri: 'Friday',
+  sat: 'Saturday',
+  sun: 'Sunday',
+);
+
+.page__body,
+.calendar__day {
+  box-sizing: border-box;
+}
+
+.page__body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+  padding: 10px;
+  display: flex;
+}
+
+.calendar__day {
+  background-color: #eee;
+  width: 100px;
+  height: 100px;
+  border: 1px solid black;
+  justify-content: center;
+  align-items: center;
+  transition-duration: 0.5s;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.calendar__day:hover {
+  cursor: pointer;
+  background-color: #ffbfcb;
+  transition-duration: 0.5s;
+  transform: translateY(-20px);
+}
+
+.calendar__day:first-child {
+  margin-left: 606px;
+}
+
+@for $i from 1 through $days {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+    font-family: Aria, sans-serif;
```

Comment:
There is a typo in the font-family declaration. It should be 'Arial' instead of 'Aria'.